src/test/scala/io/iohk/ethereum/blockchain/sync/BlockBroadcastSpec.scala

```diff
@@ -117,7 +122,7 @@ class BlockBroadcastSpec extends AnyFlatSpec with Matchers {
     override val blockBroadcast = new BlockBroadcast(etcPeerManagerProbe.ref, updatedConfig)
 
     val blockHeader: BlockHeader = baseBlockHeader.copy(number = initialPeerInfo.maxBlockNumber + 1)
-    val newBlock = NewBlock(Block(blockHeader, BlockBody(Nil, Nil)), initialPeerInfo.totalDifficulty + 1)
+    val newBlock = NewBlock(Block(blockHeader, BlockBody(Nil, Nil)), initialPeerInfo.chainWeight.increase(blockHeader))
```

Comment:
Minor: doesn't this change alter the behaviour of the test?

Reply:
No, it shouldn't, why?

Reply:
Oh nvm, the totaldifficulty is increased and by a higher amount